pkgs/games/minetest/default.nix

```diff
@@ -128,6 +129,10 @@ stdenv.mkDerivation (finalAttrs: {
     mv $out/minetest.app $out/Applications
   '';
 
+  passthru.updateScript = gitUpdater {
+    ignoredVersions = "^[^.]+$|.*-android$";
+  };
```

Comment:
Does this actually work with fetchFromGitHub? I think it needs an `url` parameter

Reply:
Yes, it does work with fetchFromGitHub, because the meta.homepage url for the src gets set to the repos base url which supports git interactions.